<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openGLparams</class>
 <widget class="QWidget" name="openGLparams">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Normal distribution generator tests</string>
  </property>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>320</y>
     <width>109</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate &gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>501</width>
     <height>126</height>
    </rect>
   </property>
   <property name="title">
    <string>Behavioral values</string>
   </property>
   <widget class="QComboBox" name="aggloTemplate">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Normal agglomeration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aggresive agglomeration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Relaxed agglomeration</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="driftTemplate">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Normal drift</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fast drift</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slow drift</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No drift</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ultra fast drift</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="decayPace">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>75</y>
      <width>396</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="l6_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Decay</string>
    </property>
   </widget>
   <widget class="QLabel" name="l6_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fc0107;&quot;&gt;No decay&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l6_12">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>95</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
   </widget>
   <widget class="QLabel" name="l6_13">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>95</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Slow</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="elementsPerStep">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>165</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="l3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total elements per step</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="angleInc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="l3_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle increase</string>
   </property>
  </widget>
  <widget class="QLabel" name="l3_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>215</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Variance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="variance">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>215</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>501</width>
     <height>76</height>
    </rect>
   </property>
   <property name="title">
    <string>Display options</string>
   </property>
   <widget class="QCheckBox" name="displayBaseline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>246</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baseline</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="displayDriftingFront">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>196</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drifting front components</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="sigmaIndexUsage">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>186</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Sigma indexing tree</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sigmaIndexNeighborhood">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>5</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>elementsPerStep</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>aggloTemplate</tabstop>
 </tabstops>
 <resources>
  <include location="../SHCCharting/SHCResources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
